tests/Examples/InputObject.hs

```diff
@@ -26,26 +25,21 @@ root :: Handler IO Query
 root = pure (\dogStuff -> pure (show dogStuff))
 
 -- $setup
--- >>> import qualified GraphQL.Internal.Encoder as Encode
--- >>> import GraphQL.Value (valueToAST)
+-- >>> import Data.Aeson (encode)
+-- >>> import GraphQL.Value.ToValue (ToValue(..))
+
+-- TODO: jml thinks it's a bit confusing to have `show` output in these
```

Comment:
Will merge this change as-is, but please do let me know if you want it changed again.